README.md

```diff
@@ -72,3 +72,24 @@ If this is the first time production has been deployed run:
 make rollout-prod
 ```
 
+## Access 
+
+In order to access the ChatBot Training Data, you need to be logged in to the cluster from your terminal. 
+
+To access the API on local machine, Start thr proxy to the cluster, use command:
+```
+make proxy
+```
+
+In order to check if the proxy is running and cluster can be accessed, use:
+```
+make get-health
+```
+
+To access API, call the URL on the browser or Postman :
+```
+http://localhost:8001/api/v1/namespaces/kscout/services/${ENV}-chatbot-training-data:http/proxy/<ENDPOINT>
+``` 
+
```

Comment:
Put the `<ENV>` and `<ENDPOINT>` in code grave marks.

Also pick a templating style. Above it was `${ENV}` but below its <ENV>` and `<ENDPOINT>`.